### DISCUSSION TITLE
Consultas TPA3

### DISCUSSION THREAD

**Discussion by rodrisanz** [dds-jv/foro#150] [TPA - 2020] [answered]
Buenas tardes!
Tenemos unas consultas con respecto al enunciado del TP: 
1. En un futuro se van a persistir las Entidades? Esto es debido a que se esta evaluando considerar tener un repositorio de esta clase. Esto aplica también para las categorías.
2. Cuales son las entidades que puede realizar egresos, las jurídicas, las base, o ambas?
3. Una entidad puede tener mas de una categoría que afecte su comportamiento, o solo va a tener una?

Desde ya muchas gracias!
Saludos.

**Comment by gastonprieto** [comment, marked as answer]
> En un futuro se van a persistir las Entidades? Esto es debido a que se esta evaluando considerar tener un repositorio de esta clase. Esto aplica también para las categorías.

Si, pero no entiendo que relacionan tiene esto con los repositorios o porque te condicionaría?

> Cuales son las entidades que puede realizar egresos, las jurídicas, las base, o ambas?

Ambas

> Una entidad puede tener mas de una categoría que afecte su comportamiento, o solo va a tener una?

Una categoría sola, pero la categoría puede tener distintos comportamientos. https://github.com/dds-jv/foro/issues/48

**Comment by rodrisanz** [comment]
Muchas gracias!
